config/config.go

```diff
@@ -1103,6 +1104,13 @@ func getSubnetConfigsFromDir(v *viper.Viper, subnetIDs []ids.ID) (map[ids.ID]sub
 }
 
 func getDefaultSubnetConfig(v *viper.Viper) subnets.Config {
+	subnetDefaults := getPrimaryNetworkConfig(v)
+	// L1s (other than Primary Network) should let ACP-226 handle block delay.
+	subnetDefaults.ProposerMinBlockDelay = acp226.MinDelayMilliseconds * time.Millisecond
```

Comment:
The MinBlockDelay should be kept to second granularity (because the proposervm timestamps are granular to the second).

So rather then setting to this time, I think we should just set it to 0.